components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/dto/TierDTO.java

```diff
@@ -23,6 +23,12 @@
   
   private String description = null;
   
+  public enum TierLevelEnum {
+     api,  application,  resource, 
```

Comment:
Please note this is part of a generated code from swagger-code-gen tool. Anyway this can be fixed  if we rename the ENUMs that are defined in the swagger definition to upper case before generating code. But I feel, if we rename this to upper case we lose some consistency as this will become the only place we are using upper case letters to define a resource.

As this is generate code, we need to rename the ENUMs for tier-level parameter in swagger definition to upper case and regenerate code to make this upper case. If we use toLowerCase() after that, that makes the user to invoke the resource using lower case name of the resource; e.g GET /tiers/api, but the swagger definition contains the upper case name of the resource which induces that the resource should be invoked using GET /tiers/API which will be failed.